resources/allowlist.gni

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Allowlist file used by brave resources, created after merging chrome allowlist
+# and brave resources.
+brave_allowlist_file = "${root_gen_dir}/brave_resources_allowlist.txt"
+
+template("brave_allowlist") {
```

Comment:
why are you not using the `generate_resource_allowlist` target template?

Reply:
actually why do we need a template for this at all? Just define an action called `merge_allowlists` (probably in `brave/BUILD.gn` and it will depend on the `generate_resource_allowlist` in wireguard (where the output filename will also be defined). In the future if we need to add anything else, we will have additional `generate_resource_allowlist` targets that are also deps of the `merge_allowlist` action

Reply:
done